js-green-licenses.json

```diff
@@ -10,24 +10,21 @@
 		"BSD",
 		"BSD-2-Clause",
 		"BSD-3-Clause",
-		"CC0-1.0"
+		"CC0-1.0",
+		"CC-BY-3.0"
 	],
 	"packageWhitelist": [
 		"@eventespresso/react-exit-modal-typeform",
-		"@wordpress/jest-preset-default",
 		"js-green-licenses",
 		"@typeform/embed",
 		"scrollbar-width",
 		"webfontloader",
 		"validate-npm-package-license",
 		"spdx-correct",
-		"spdx-exceptions",
```

Comment:
were these removed on purpose? I don't see why they're being removed

Reply:
otherwise these changes seem fine

Reply:
the were removed because they no longer need to be whitelisted.  The license validation now correctly passes on these packages.

Reply:
ahhhh- and they pass now because you added a new "greenLicense". Makes sense!